facia-json/src/main/scala/com/gu/facia/client/models/Config.scala

```diff
@@ -54,6 +54,11 @@ object Metadata extends StrictLogging {
   }
 }
 
+object DisplayHints {
+  implicit val jsonFormat = Json.format[DisplayHints]
+}
+
+case class DisplayHints(maxItemsToDisplay: Option[Int])
```

Comment:
Do we need this as an `Option`? I guess we would if we added more to `DisplayHints`.